uber/models.py

```diff
@@ -892,6 +892,12 @@ class RoomAssignment(MagModel):
 class NoShirt(MagModel):
     attendee_id = Column(UUID, ForeignKey('attendee.id'), unique=True)
 
```

Comment:
Since we have two different foreign keys into the same table, we use the ``primaryjoin`` parameter.

There are no backrefs currently since we don't currently display this info anywhere, it's mostly just there in case of a dispute or something.